.changeset/slimy-donuts-report.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': minor
+---
+
+Add support for non-responsive values to `Grid`'s `gap`, `columns`, and `areas` props.
```

Comment:
```suggestion
Added support for non-responsive values to `Grid`'s `gap`, `columns`, and `areas` props.
```

Reply:
Whoops, this changeset is meant for [the other PR](https://github.com/Shopify/polaris/pull/10489) - have removed it 👍